### PR TITLE
Injecting event manager to allow listeners to trigger correctly

### PR DESCRIPTION
**Pull request by markchalloner**

### CODE REVIEW

File: src/DoctrineMongoODMModule/Service/ConnectionFactory.php [hunk 1]
```diff
@@ -83,7 +83,10 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
             $configuration->setDefaultDB($dbName);
         }
 
-        return new Connection($connectionString, $options->getOptions(), $configuration);
+        /** @var $configuration \Doctrine\Common\EventManager */
+        $eventManager = $serviceLocator->get('doctrine.eventmanager.' . $this->getName());
```
Comment (malarzm) [inline src/DoctrineMongoODMModule/Service/ConnectionFactory.php]: This should be `$container` I believe

**Comment by markchalloner** [comment]
Any chance of this being merged?

**Comment by Bombdog** [comment]
@markchalloner Mine is a month older than yours, not much happening around here

**Comment by alcaeus** [comment]
Please give us a little more time - @malarzm and I now have push access to this repository and will be taking care of issues and PRs soon.

**Comment by alcaeus** [comment]
@markchalloner can you rebase on top of master and squash the two commits? Thanks!

**Comment by markchalloner** [comment]
Done. Cheers

**Comment by alcaeus** [comment]
Thank you, and again, sorry for the delay!

**Pull request merged**